src/main/java/pageobject/TestSessionPage.java

```diff
@@ -0,0 +1,208 @@
+			System.out.println("TestSessions link not found");
+			throw (e);
+		}
+	}
+	
+		
+	public void btn_AddTestSession() throws Exception {
+		try {
+			//Thread.sleep(2000);
+			driver.manage().timeouts().implicitlyWait(25, TimeUnit.SECONDS);
+			clickElement(AddTestSession);			
+			System.out.println("AddTestSession button clicked");
+		}catch (Exception e) {
+			System.out.println("AddTestSession button not found");
+			throw (e);
+		}
+	}
+	
+	
+	public void dropDown_SelectCampaign() throws Exception {
+		try {			
+			clickElement(SelectCampaign);
+			System.out.println("SelectCampaign drop-down open");			
+			clickElement(selectCampaignValue);
+			System.out.println("Campaign Value selected");
+		}catch (Exception e) {
+			System.out.println("SelectCampaign drop-down not found");
+			throw (e);
+	}
+  }
+	
+	
+	public void select_ExpirationDate() throws Exception {
+		try {			
+			clickElement(ExpirationDate);
+			System.out.println("ExpirationDate selected");
+			clickElement(todayDate);
+			System.out.println("todayDate selected");
+		}catch (Exception e) {
+			System.out.println("ExpirationDate not selected");
+			throw (e);
+	}
+  }
+	
+	public void select_Numbers() throws Exception {
+		try {			
+			clickElement(Numbers);
+			System.out.println("Numbers selected");			
+		}catch (Exception e) {
+			System.out.println("Numbers not selected");
+			throw (e);
+	}
+  }
+	
+	public void enter_SessionData() {		
+		try {			
+			//Thread.sleep(1000);
+			driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+			clickElement(EnterSessionData);
+			FileReader reader = new FileReader("Binaries/SampleJson.json");			
+			JSONParser jsonpaser = new JSONParser();			
+			JSONObject jsonObject = (JSONObject) jsonpaser.parse(reader);
+			System.out.println(jsonObject.toJSONString());
+			driver.findElement(EnterSessionData).sendKeys(jsonObject.toJSONString());			
+		}catch (Exception e) {
+			System.out.println(e.toString());
+			e.printStackTrace();
+		}
+		
+	}
+	
+	public void btn_Create_Method() {		
+		try {			
+			clickElement(CreateButton);			
+			System.out.println("Create Button clicked......");
+		}catch (Exception e) {
+			System.out.println("Create Button not clicked......");
+			e.printStackTrace();
+		}
+		
+	}
+	
+	public void successPopUp_Method() {		
+		try {			
+			//Thread.sleep(2000);
+			driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+			if (!driver.findElement(Model_Msg).getText().equals("Error!")) {
+				System.out.println("Test Session Successflly Created......"+ driver.findElement(Model_Msg).getText());
+				//Thread.sleep(1000);
+				driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+				clickElement(SuccessPopUp_CloseBtn);
+		    	System.out.println("SuccessPopUp_Close Btn clicked......");				
+		     }else {		    	 
+		    	 System.out.println(driver.findElement(Model_Msg).getText()+"--------"+ driver.findElement(PopUp_TxtMsg).getText());
+		    	 Assert.fail();
```

Comment:
Assertion should be written inside step definition in Then statement.
You can written the element value from page objects